src/main/java/org/prebid/server/bidder/improvedigital/ImprovedigitalBidder.java

```diff
@@ -84,6 +131,10 @@ private void parseAndValidateImpExt(Imp imp) {
     private HttpRequest<BidRequest> resolveRequest(BidRequest bidRequest, Imp imp) {
         final BidRequest modifiedRequest = bidRequest.toBuilder()
                 .imp(Collections.singletonList(imp))
+                .user(bidRequest.getUser() != null ? bidRequest.getUser().toBuilder()
```

Comment:
Use 121 character length to it's fulliness, please :)
`.user(bidRequest.getUser() != null`
` ? bidRequest.getUser().toBuilder().ext(getAdditionalConsentProvidersUserExt(bidRequest)).build()`
` : null)`